### PR TITLE
Improve logging for 5.3.1-cp7

### PR DESCRIPTION
**Pull request by gharris1727**
This cherry-picks the changes from #364 to be used for performance tuning.

### CODE REVIEW

File: src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java [hunk 1]
```diff
@@ -143,6 +143,7 @@ public void start(Map<String, String> props, ElasticsearchClient client) {
 
       writer = builder.build();
       writer.start();
+      log.info("Elasticsearch sink task will {} records with null values", behaviorOnNullValues);
```
Comment (levzem) [inline src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java]: we know this value from the configs - is this superfluous

File: src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java [hunk 2]
```diff
@@ -235,12 +235,16 @@ public ElasticsearchWriter build() {
   }
 
   public void write(Collection<SinkRecord> records) {
+    log.debug("Writing {} records to Elasticsearch", records.size());
```
Comment (levzem) [inline src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java]: duplicates the `put()` log message considering `put()` calls `write()`

File: src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java [hunk 3]
```diff
@@ -235,12 +235,16 @@ public ElasticsearchWriter build() {
...
     for (SinkRecord sinkRecord : records) {
+      log.trace("Writing record to Elasticsearch: topic/partition/offset {}/{}/{}",
```
Comment (levzem) [inline src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java]: does `log.isTraceEnable()` make a difference?

File: src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java [hunk 4]
```diff
@@ -222,40 +222,58 @@ private Version getServerVersion() throws IOException {
...
     } else if (esVersion.startsWith("7.")) {
-      return Version.ES_V7;
+      version = Version.ES_V7;
+      log.info("Detected Elasticsearch version is {}", version);
+    } else {
+      version = defaultVersion;
+      log.info("Detected unexpected Elasticsearch version {}, using {}", esVersion, version);
     }
-    return defaultVersion;
+    return version;
```
Comment (levzem) [inline src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java]:
```
if (version != defaultVersion) {
     log.info(...)
}
```

Comment (gharris1727) [reply src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java]: Declined after discussing it on the main PR.

**Comment by gharris1727** [comment]
This cherry-pick is no longer necessary as we will not be releasing 5.3.1-cp7.

**Pull request closed without merging**